packages/react-native-jigsaw/src/components/RadioButton/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react";
+import { View, StyleProp, ViewStyle } from "react-native";
+import {
+  COMPONENT_TYPES,
+  FORM_TYPES,
+  FIELD_NAME,
+  createTextProp,
+  createTextStyle,
+} from "../../core/component-types";
+import themeT from "../../styles/DefaultTheme";
+import { radioButtonGroupContext } from "./context";
+
+export interface RadioButtonGroupProps {
+  direction?: "horizontal" | "vertical";
```

Comment:
can you make this an enum? You can do this:

```
enum Direction {
Horizontal = "horizontal",
Vertical = "vertical"
}
```

and i believe it will convert that string correctly to a lower case string